lib/js/src/protocol/_SdlProtocolBase.js

```diff
@@ -268,6 +270,33 @@ class _SdlProtocolBase {
         return this._messageID++;
     }
 
+    /**
+     * Gets the Vehicle details received in StartService ACK protocol message
+     * @returns {VehicleType|null} - A RPC VehicleType struct received from the packet if exists null otherwise.
+     */
+    _getVehicleType(sdlPacket) {
+        const make = sdlPacket.getTag(_ControlFrameTags.RPC.StartServiceACK.VEHICLE_MAKE);
+        const model = sdlPacket.getTag(_ControlFrameTags.RPC.StartServiceACK.VEHICLE_MODEL);
+        const modelYear = sdlPacket.getTag(_ControlFrameTags.RPC.StartServiceACK.VEHICLE_MODEL_YEAR);
+        const trim = sdlPacket.getTag(_ControlFrameTags.RPC.StartServiceACK.VEHICLE_TRIM);
+
+        // TODO: awaiting 0293 revisions to process those fields
+        const systemHardwareVersion = sdlPacket.getTag(_ControlFrameTags.RPC.StartServiceACK.VEHICLE_SYSTEM_HARDWARE_VERSION);
+        const systemSoftwareVersion = sdlPacket.getTag(_ControlFrameTags.RPC.StartServiceACK.VEHICLE_SYSTEM_SOFTWARE_VERSION);
```

Comment:
This method seems unfinished since these values are never used.

Reply:
@renonick87 no, the method is finished and reflects the current state of the proposal. Without revisions, we receive those fields here but never use or process them. The TODO was added to reflect that the processing needs to be added after accepting the revisions.